packages/terra-hookshot/tests/wdio/hookshot-combinations-spec.js

```diff
@@ -0,0 +1,892 @@
+/* global browser, Terra, before */
+
+describe('[1] Display correctly for contentAttachment="top start" & targetAttachment="top start"', () => {
+  before(() => browser.setViewportSize(Terra.viewports('small')[0]));
```

Comment:
We should be able to add a top-layer describe to set the screen size. and then simplify these like:
```
describe('Hookshot', () => {
    before(() => browser.setViewportSize(Terra.viewports('small')[0]));

    describe('contentAttachment="top start"', () => {
        before(() =>  browser.url('/#/raw/tests/hookshot/hookshot-content-attachment-ts'));

        it('triggers targetAttachment="top start"', () => {
            browser.click('#attach-TS');   
            browser.waitForVisible('#attachment-content');
       });
      Terra.should.matchScreenshot('targetAttachment="top start"', { selector: '#attachment-bounds' });

        it('triggers targetAttachment="top center"', () => {
            browser.click('#attach-TC');   
            browser.waitForVisible('#attachment-content');
       });
      Terra.should.matchScreenshot('targetAttachment="top center"', { selector: '#attachment-bounds' });
    });
})
```